packages/studio/src/utils/codemirror.utils.ts

```diff
@@ -31,13 +29,15 @@ export const myTheme = EditorView.theme({
     }
 }, {dark: true})
 
-export const language = new Compartment 
+export const language = new Compartment
 
 export const myHighlightStyle = HighlightStyle.define([
-  {tag: customTags.chord, color: "#0481aa"},
+  {tag: tags.keyword, color: "#0481aa"},
   {tag: tags.bracket, color: "#fc6"},
```

Comment:
I think we should be able to style brackets around chords independently from other elements.

Right now this tag for bracket doesn't do anything, the bracket color comes from line 10 which is the color for everything that doesn't match a tag. 

So I think we need something in the grammar for the chord brackets specifically. I say chord brackets specifically because ideally I think the meta tag brackets would be the same grey color as the directive value. But the chord brackets would be a slightly transparent version of whatever the chord color is.

Reply:
Good call. Fixed in a420ac3cd914ce6b32d40688540b8dd5c9377402

<img width="545" alt="image" src="https://user-images.githubusercontent.com/173/187774533-d782e506-cddb-4557-a133-1085acd9f3de.png">